.gitignore

```diff
@@ -6,6 +6,12 @@
 /public/mix-manifest.json
 /storage/*.key
 /vendor
+/resources/lang/de
+/resources/lang/fr
+/resources/lang/gr
+/resources/lang/it
+/resources/lang/la
+/resources/lang/sp
```

Comment:
Removed the language files that is not hungarian or english (users can not modify those).
The motivation was that changing the files on the server side could cause us problems, so we can manage the language and hungarian translations here, and the users can manage the rest based on our work, separately.
But these files have to be deleted from git to .gitignore take effect and that would erase our work so far (eg. latin translations).

Reply:
Can we import these into the database? Maybe we could run some script?

Reply:
Hmm... Do you mean add the existing translations to this new table? Maybe. But the script have to run after addig the database and before deleting the files. And where to place the script? We would need this only once. Should we do it independently of git?

Reply:
The files won't be deleted if we pull this PR, they will be simply ignored by git. You could add it maybe as an artisan command since we already have most of the stuff there and then we will remove in a later PR.

Reply:
Okay. And we will remove the files later, right?